templates/server/fastcgi_params.erb

```diff
@@ -5,12 +5,13 @@ fastcgi_param  REQUEST_METHOD     $request_method;
 fastcgi_param  CONTENT_TYPE       $content_type;
 fastcgi_param  CONTENT_LENGTH     $content_length;
 
-fastcgi_param  SCRIPT_FILENAME    $request_filename;
```

Comment:
It does seem like this line doesn't exist in current upstream, though there is `SCRIPT_NAME`.